streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/ClientState.java

```diff
@@ -50,10 +49,17 @@
     private final Set<TaskId> prevStandbyTasks;
 
     private final Map<String, Set<TaskId>> consumerToPreviousStatefulTaskIds;
+    // the following four maps are used only for logging purposes;
+    // TODO: we could consider merging them with other book-keeping maps
+    private final Map<String, Set<TaskId>> consumerToPreviousActiveTaskIds;
+    private final Map<String, Set<TaskId>> consumerToAssignedActiveTaskIds;
+    private final Map<String, Set<TaskId>> consumerToAssignedStandbyTaskIds;
+    private final Map<String, Set<TaskId>> consumerToRevokingActiveTaskIds;
```

Comment:
We should add an empty line here to avoid mixing with the other maps with production use cases.